lib/auth/auth_with_roles_test.go

```diff
@@ -4245,13 +4258,13 @@ func TestListUnifiedResources_WithSearch(t *testing.T) {
 		SortBy:         types.SortBy{IsDesc: true, Field: types.ResourceMetadataName},
 	})
 	require.NoError(t, err)
-	require.Len(t, resp.Resources, 2)
+	require.Len(t, resp.Resources, 1)
```

Comment:
can we filter over type instead of a search keyword? that might be more future-proof (line 4256)

Reply:
we have a "kinds" filter test already, this test is for using "search", i figured i could feed two birds with one scone by searching and making sure it can return SAML apps